src/main/java/frc/robot/notification/SimLedIO.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.notification;
+
+public class SimLedIO implements ILedIO{
+
+  private double ledSparkValue = 0;
+  
+  public SimLedIO(){}
+
+  @Override
+  public void setSpark(double sparkValue){
+    ledSparkValue = sparkValue;
```

Comment:
If we do the above comment then this can make a `Mechanism2d` on the Sim GUI and change its color to show for the simulator